### PR TITLE
Add motoko echo example

### PR DESCRIPTION
**Pull request by dansteren**
This implements the Motoko echo example found here: https://github.com/dfinity/examples/tree/master/motoko/echo, with a single test (since this is a straightforward example).

Resolves https://github.com/demergent-labs/azle/issues/56

### CODE REVIEW

File: examples/motoko_examples/echo/test/test.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    {
+        name: 'deploy',
+        prep: async () => {
+            execSync(`dfx deploy`, {
+                stdio: 'inherit'
+            });
+        }
+    },
+    {
+        name: 'say',
```
Comment (lastmjs) [inline examples/motoko_examples/echo/test/test.ts:39]: Let's just put a few more tests in here

**Review by lastmjs — CHANGES_REQUESTED** [review]

**Comment by lastmjs** [comment]
Remember to put the notices and licenses in here

File: examples/motoko_examples/echo/test/test.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        test: async () => {
+            const phrase = `
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+                enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
+                in reprehenderit in voluptate velit esse cillum dolore eu fugiat
+                nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+                sunt in culpa qui officia deserunt mollit anim id est laborum.
+            `
```
Comment (lastmjs) [inline examples/motoko_examples/echo/test/test.ts:71]: semicolon

File: examples/motoko_examples/echo/test/test.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+            return {
+                ok: result === phrase
+            };
+        }
+    },
+    {
+        name: 'say nothing',
+        test: async () => {
+            const phrase = ""
```
Comment (lastmjs) [inline examples/motoko_examples/echo/test/test.ts:52]: Single quotes in JS, and semicolon

File: examples/motoko_examples/echo/test/test.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        prep: async () => {
+            execSync(`dfx deploy`, {
+                stdio: 'inherit'
+            });
+        }
+    },
+    {
+        name: 'say',
+        test: async () => {
+            const phrase = "This is a test."
```
Comment (lastmjs) [inline examples/motoko_examples/echo/test/test.ts:41]: Single quotes in JS, and semicolon

**Review by lastmjs — CHANGES_REQUESTED** [review]

**Review by lastmjs — APPROVED** [review]

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -67,6 +67,7 @@ jobs:
             "examples/management_canister",
             "examples/motoko_examples/calc",
             "examples/motoko_examples/counter",
+            "examples/motoko_examples/echo"
```
Comment (lastmjs) [inline .github/workflows/test.yml:70]: Needs a comma

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 5 above)
Comment (dansteren) [reply .github/workflows/test.yml]: Added missing comma.

**Review by lastmjs — APPROVED** [review]

**Pull request merged**